md_template/model.md

```diff
@@ -0,0 +1,51 @@
+This is a simple template for setting up automatic report generation with GPkit.
+To generate this file, run
+```
+make model.pdf
+```
+in this directory.
+
+\section{Example Model}
```

Comment:
Use `#Example Model` instead so that the markdown also knows that it's a header.